e2e/rbd.go

```diff
@@ -373,31 +373,30 @@ var _ = Describe("RBD", func() {
 			}
 			By("verify generic ephemeral volume support", func() {
 				// generic ephemeral volume support is supported from 1.21
-				if !k8sVersionGreaterEquals(f.ClientSet, 1, 21) {
-					Skip("generic ephemeral volume only supported from v1.21+")
-				}
-				// create application
-				app, err := loadApp(appEphemeralPath)
-				if err != nil {
-					e2elog.Failf("failed to load application: %v", err)
-				}
-				app.Namespace = f.UniqueName
-				err = createApp(f.ClientSet, app, deployTimeout)
-				if err != nil {
-					e2elog.Failf("failed to create application: %v", err)
-				}
-				// validate created backend rbd images
-				validateRBDImageCount(f, 1, defaultRBDPool)
-				err = deletePod(app.Name, app.Namespace, f.ClientSet, deployTimeout)
-				if err != nil {
-					e2elog.Failf("failed to delete application: %v", err)
-				}
-				// validate created backend rbd images
-				validateRBDImageCount(f, 0, defaultRBDPool)
-				// validate images in trash
-				err = waitToRemoveImagesFromTrash(f, defaultRBDPool, deployTimeout)
-				if err != nil {
-					e2elog.Failf("failed to validate rbd images in pool %s trash: %v", defaultRBDPool, err)
+				if k8sVersionGreaterEquals(f.ClientSet, 1, 21) {
```

Comment:
return cannot be placed in `By`/`It` block. updated E2E to make use of `Ginkgo.It` instead of `Ginkgo.By`. if we use `ginkgo.It` we can skip the tests the same is not possible in `ginkgo.By`

Reply:
Ah, that's much better!

Reply:
@nixpanic I reverted back the changes as moving it `It` won't work. currently, this is the easy solution to run E2E for now. but we need to find an alternative way for this one. IMO  this is fine for now instead of skipping the whole test suite.